rust/processor/src/models/user_transactions_models/signatures.rs

```diff
@@ -353,6 +354,12 @@ impl Signature {
                 "".to_string()
             },
         };
+        let signature_bytes = match signature.signature.as_ref().unwrap() {
```

Comment:
This isn't mergeable on main right? b/c we don't want to unwrap here.

Reply:
nope this is a hack; good call out and thanks Rustie marking this as draft

Reply:
Also why are we doing this?

Reply:
existing processor is broken.

context: randomnet aptos-core cut is from latest main with a lot of cherrypicks. To use that fullnode, we are required to bump protobuf versions; otherwise, processor cannot recognize the new transaction type.